tests/test_configator.py

```diff
@@ -0,0 +1,21 @@
+"""This module tests the configator module."""
```

Comment:
Although this docstring claims this module tests the entire `configator` module, it in fact only tests the `generate_build_gradle` file. This module should be renamed to a more appropriate name, and this docstring updated.

Reply:
Agreed @Michionlion. If there are time constraints, then please update the filename and docstring next string. Otherwise, if you have the time, this is a worthwhile change to make.